src/Home.tsx

```diff
@@ -211,10 +181,10 @@ const Home = ({
 												value={lifetimeData}
 												disabled={isLoading}
 											>
-												<option value="12">12 epochs (hours)</option>
-												<option value="24">24 epochs (1 day)</option>
-												<option value="48">48 epochs (2 days)</option>
-												<option value="96">96 epochs (4 days)</option>
+												<option value="12h">12 hours (~12 epochs)</option>
+												<option value="24h">1 day (~24 epochs)</option>
+												<option value="48h">2 days (~48 epochs)</option>
+												<option value="96h">4 days (~96 epochs)</option>
```

Comment:
Separate matter. We also have _exact_ number of epochs, but approximate number of hours.